index.js

```diff
@@ -1,40 +1,59 @@
 'use strict';
 
+const EMBER_VERSION_WITH_DEPRECATED_COMPONENT_DOLLAR = '3.0.0'; // @todo replace with real version
```

Comment:
This needs to be replaced with the real version, that will introduce the deprecation, so the addon will override the (deprecated) implementation with its own without any deprecation warning.

Reply:
done